Src/TestFramework/Shared/AssemblyInfo.cs

```diff
@@ -2,16 +2,16 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyCompany("Microsoft")]
-[assembly: AssemblyCopyright("Copyright © Microsoft. All Rights Reserved.")]
-
-[assembly: ComVisible(false)]
+#pragma warning disable CS0436
```

Comment:
is there a matching 
`#pragma warning enable`
...at the bottom of this file?

Reply:
added, thanks